src/cijoe/core/transport.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, config, output_path):
         """Initialize the CIJOE SSH Transport"""
 
         self.config = config
-        self.shell = self.config.options.get("run").get("shell", "sh")
+        self.shell = self.config.options.get("run", {}).get("shell", "sh")
```

Comment:
So, I know I initially mused on adding a top-level ``run`` item. That that I have seen this and mulled a bit on it, then it would be great if this was "top-level" under a ``cijoe`` item. Then, going forward we can eventually move the "transport" section into this as well.
By doing so, then the config-items for cijoe is the same as for everything else. So, for now we would have:

```yaml
cijoe:
  run:
    shell: bash
  env:
    foo: "this is so 42"
```

Reply:
This has been fixed now :)